hub/models.py

```diff
@@ -99,9 +100,9 @@ def clean(self):
         try:
             self.user_matches(user, ignore_invalid_queries=False)
         except FieldError as e:
-            raise ValidationError({'user_queries': e.message})
+            raise serializers.ValidationError({'user_queries': e.message})
         except TypeError:
-            raise ValidationError(
+            raise serializers.ValidationError(
```

Comment:
looks like the django admin interface won't tolerate this :( it returns a 500 instead of showing the validation message. this isn't (yet) used in any DRF view, so the old way should be ok.